src/schemas/validation-login-schema.ts

```diff
@@ -0,0 +1,18 @@
+import * as zod from 'zod';
```

Comment:
Porque não fazer como na documentação?
https://zod.dev/?id=basic-usage

Não que seu código não funciona, fora questão de convenção, vc tá importando tudo, pode causar enganos ao tentar usar algo que não era o queria. Sua opção funcionaria melhor se vc fosse utilizar algo fora do { z } que está no export *. Mas tudo pelo que vi está no "z".

```suggestion
import { z } from 'zod';
```

ou 
```suggestion
import { z as zod } from 'zod';
```

Reply:
Realmente, Lucas. A importação foi modificada.